### ISSUE TITLE
Using custom VoxelGeneratorScript in editor's voxelgraph?

### ISSUE THREAD

**Issue by Shadowblitz16** [Zylann/godot_voxel#557]
Is there a way to register a VoxelGeneratorScript so I can use it in the editor and in the VoxelGeneratorGraph?

**Comment by Zylann** [comment]
No, graphs work in a different way compared to generators, it's not possible to nest them like this.
